src/main/java/duke/WrongTask.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * Represents an Exception due to user inputting a wrong task keyword.
+ */
+public class WrongTask extends Exception {
```

Comment:
Would it be better to centralize a DukeExceptions class which can then be inherited by the various separate Exception classes?